service/internal/unittest/default_cluster.go

```diff
@@ -42,6 +44,24 @@ func DefaultCluster() infrastructurev1alpha2.AWSCluster {
 			},
 		},
 		Status: infrastructurev1alpha2.AWSClusterStatus{
+			Cluster: infrastructurev1alpha2.CommonClusterStatus{
+				Conditions: []infrastructurev1alpha2.CommonClusterStatusCondition{
+					{
+						LastTransitionTime: metav1.NewTime(time.Now().Add(-90 * time.Minute)),
+						Condition:          "Creating",
+					},
+					{
+						LastTransitionTime: metav1.NewTime(time.Now().Add(-60 * time.Minute)),
+						Condition:          "Created",
+					},
+					{
+						LastTransitionTime: metav1.NewTime(time.Now().Add(-15 * time.Minute)),
+						Condition:          "Updating",
+					},
```

Comment:
I think the order should be reversed. AFAIK the latest condition is put at the top. 

```
$ k get -o yaml awsclusters.infrastructure.giantswarm.io 6rffh
...
status:
  cluster:
    conditions:
    - condition: Created
      lastTransitionTime: "2020-08-18T11:46:02Z"
    - condition: Creating
      lastTransitionTime: "2020-08-18T11:25:04Z"
...
```

Reply:
You're right I'll update it so it matches our expections